packages/package_info/Move.toml

```diff
@@ -3,7 +3,7 @@ name = "package_info"
 edition = "2024.beta"
 
 [dependencies]
-Sui = { git = "https://github.com/MystenLabs/sui.git", subdir = "crates/sui-framework/packages/sui-framework", rev = "framework/mainnet", override=true }
+Sui = { git = "https://github.com/MystenLabs/sui.git", subdir = "crates/sui-framework/packages/sui-framework", rev = "framework/testnet", override=true }
```

Comment:
Why is override true here?

Reply:
This is a classic workaround to get around "Sui" versioning story bugs (source verification missmatch)